server/sonar-server/src/main/java/org/sonar/server/util/AbstractStoppableExecutorService.java

```diff
@@ -45,6 +45,15 @@ public AbstractStoppableExecutorService(T delegate) {
 
   @Override
   public void stop() {
+    try {
+      stopIt();
+    } catch (Throwable e) {
+      e.printStackTrace();
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Use a logger to log this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1148)